src/EA.Weee.Web/Areas/AATFReturn/ViewModels/NonObligatedValuesViewModel.cs

```diff
@@ -17,14 +18,18 @@ public class NonObligatedValuesViewModel
 
         public bool Dcf { get; set; }
 
+        private ICategoryValueTotalCalculator categoryValueCalculator;
+
         public NonObligatedValuesViewModel()
```

Comment:
Should be able to pass ICategoryValueTotalCalculator as constructor parameter rather than instantiate in the constructor